src/slurm_plugin/clustermgtd.py

```diff
@@ -304,6 +305,11 @@ def _get_terminate_config(self, config):
         self.insufficient_capacity_timeout = config.getfloat(
             "clustermgtd", "insufficient_capacity_timeout", fallback=self.DEFAULTS.get("insufficient_capacity_timeout")
         )
+        self.ec2_backing_instance_max_count = config.getfloat(
```

Comment:
does it need to be a float?

Reply:
I don't think it does, but I was following the precedent above